docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -491,15 +517,115 @@ If other CDDL representation types will be used in future versions of
+| `family_name_birth` | `birth_family_name` | |
+| `given_name_birth` | `birth_given_name` | |
+| `birth_place` | `place_of_birth/locality` | * |
+| `birth_country` | `place_of_birth/country` | |
+| `birth_state` | `place_of_birth/region` | |
+| `birth_city` | `place_of_birth/locality` | |
+| `resident_address` | `address/formatted` | |
+| `resident_country` | `address/country` | |
+| `resident_state` | `address/region` | |
+| `resident_city` | `address/locality` | |
+| `resident_postal_code` | `address/postal_code` | |
+| `resident_street` | `address/street_address` | * |
+| `resident_house_number` | `address/house_number` | * |
+| `gender` | `gender` | Note: Data type mismatch - JWT claims use `male`, `female` and custom text values.* |
+| `nationality` | `nationalities` | Note: Defined as an array, here containing only one entry.|
+| `issuance_date` | `iat` | |
+| `expiry_date` | `exp` | |
+
+*Table 3: Data Element Identifier to Claim Name Mapping*
+
+Fields marked with (*) indicate that there is not a one-to-one mapping
+between the data element and a suitable pre-defined JWT claim.
+This should be resolved in future versions of this document.
+
+EXAMPLE: The following example shows a PID in SD-JWT VC format.
+
+```json
+{
+    "vct": "https://memberstate.example/credential/pid",
+    "vct#integrity": "sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM",
+
+    "given_name": "Jean",
+    "family_name": "Dupont",
+    "birthdate": "1980-05-23",
+
+    "age_equal_or_over": {
+        "12": true,
+        "14": true,
+        "16": true,
+        "18": true,
+        "21": true,
+        "65": false
+    },
+    "age_in_years": 44,
+    "age_birth_year": 1980,
+
+    "address": {
+        "street_address": "123 Via Appia",
+        "locality": "Rome",
+        "region": "Lazio",
+        "postal_code": "00100",
+        "country": "IT"
+    },
+
+    "nationalities": ["FR"],
+
+    "gender": "male",
+
+    "place_of_birth": {
+        "locality": "Leipzig",
+        "region": "Saxony",
+        "country": "DD"
+    },
+
+    "cnf": {
+        "jwk": {
+            "kty": "EC",
+            "crv": "P-256",
+            "x": "52aDI_ur05n1f_p3jiYGUU82oKZr3m4LsAErM536crQ",
+            "y": "ckhZ-KQ5aXNL91R8Eufg1aOf8Z5pZJnIvuCzNGfdnzo"
+        }
+    }
+}
+```
+
+Note: The `cnf` claim is used for expressing key binding in SD-JWT VCs.
+The example above shows a public key in JWK format.
+
+Note: Additional technical claims are not shown here, including
+references to the issuer, validity status information, and more.
+
+
+##### 2.6.3.3 Further stipulations
+
+SD-JWT VCs containing data elements in Table 1 and Table 2 SHALL be
+released only in a VP Token, as specified in \[OpenID4VP\]. This means
+that these data elements SHALL be signed by the PID Provider, not by the
 
 At the discretion of the PID Provider, domestic data elements (see
 section 2.2.2) MAY be released in either a VP Token or an ID Token.
```

Comment:
I guess this should be moved down?

Reply:
The sentence before is finished after this paragraph:
```
that these data elements SHALL be signed by the PID Provider, not by the

At the discretion of the PID Provider, domestic data elements (see
section 2.2.2) MAY be released in either a VP Token or an ID Token.

Wallet Instance.
```

It looks like line 621/622 should be moved to ~629?

Reply:
Thanks, will fix!